docs/dapr-sdk-workflows/legal/LICENSE

```diff
@@ -0,0 +1,347 @@
+The GNU General Public License (GPL)
```

Comment:
I thought we removed this last release?

Reply:
It comes with classpath exception from Oracle. It should be OK because we are not depending on/extending their codebase but on the compiled lib only.